src/components/common/UserProfileEdit/UserMainSettings.tsx

```diff
@@ -94,15 +94,10 @@ const UserMainSettings = ({
         },
       },
     });
+
+    updateUser?.(walletAddress, true);
   };
 
-  useEffect(
```

Comment:
With the addition of a "background update" option, updating the user on dismount is no longer necessary and can be performed in the form submit handler.